api/services/call_log_service.py

```diff
@@ -68,15 +68,26 @@ def create_call_logs(self, jsonData):
             if "flow" in jsonData and jsonData["flow"] is not None:
                 flow_name = helpers.fetch_by_key("name", jsonData["flow"])
 
+            content_version = None
             content_id = None
             if "content_id" in jsonData:
                 content_id = (jsonData["content_id"],)
                 content_data = models.Content.query.get(content_id)
+                language_id = (jsonData["language_id"],)
```

Comment:
I didn't understood the syntax. What this is doing. Also if `language_id` is not preset in jsonData the I think it will throw error of `key doesn't exist`. You can use JsonData.get("laguage_id") kind of thing here